src/adv.py

```diff
@@ -24,28 +24,43 @@
 
 # Link rooms together
 
-room['outside'].n_to = room['foyer']
-room['foyer'].s_to = room['outside']
-room['foyer'].n_to = room['overlook']
-room['foyer'].e_to = room['narrow']
-room['overlook'].s_to = room['foyer']
-room['narrow'].w_to = room['foyer']
-room['narrow'].n_to = room['treasure']
-room['treasure'].s_to = room['narrow']
+room['outside'].n = room['foyer']
+room['foyer'].s = room['outside']
+room['foyer'].n = room['overlook']
+room['foyer'].e = room['narrow']
+room['overlook'].s = room['foyer']
+room['narrow'].w = room['foyer']
+room['narrow'].n = room['treasure']
+room['treasure'].s = room['narrow']
 
 #
 # Main
 #
 
 # Make a new player object that is currently in the 'outside' room.
+player = Player("Jack", room["outside"], [])
 
 # Write a loop that:
-#
 # * Prints the current room name
 # * Prints the current description (the textwrap module might be useful here).
 # * Waits for user input and decides what to do.
-#
+while True:
+    print(f"You are currently in the {player.current_location.name}")
+    print(player.current_location.description)
+    direction = input(
+        "Which direction would you like to go? (enter n, e, s, or w):")
+    if direction == "n" or "e" or "s" or "w":
```

Comment:
This will always evaluate the true because you're saying if direction is "n" OR "e" is truthy OR "s" is truthy OR "e" is truthy. What you would want is something like `if direction == "n" or direction == "e" or direction == "s" or direction == "w"`